src/System.Private.CoreLib/src/System/DateTime.Windows.cs

```diff
@@ -0,0 +1,182 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System
+{
+    public readonly partial struct DateTime
+    {
+        internal static readonly bool s_isLeapSecondsSupportedSystem = IsLeapSecondsSupportedSystem();
```

Comment:
`isLeapSecondsSupportedSystem` does not sound right to me. What about `s_systemSupportsLeapSeconds` ? 

Then code reads better:
```c#
            if (s_systemSupportsLeapSeconds)
            {
                return InternalFromFileTime(fileTime);
            }
```